jarbas/dashboard/templatetags/dashboard.py

```diff
@@ -11,3 +13,47 @@ def rename_title(title):
     title = title.replace('modificar', 'visualizar')
     title = title.replace('Modificar', 'Visualizar')
     return title
+
+
+@register.filter()
+def percentof(amount, total):
+    try:
+        return f'{brazilian_float(amount * 100 / total)}%'
+    except ZeroDivisionError:
+        return None
+
+
+@register.filter()
+def brazilian_reais(value):
+    return f'R$ {brazilian_float(value)}'
+
+
+@register.filter()
+def brazilian_float(value):
+    # TODO properly use locales
+    value = value or 0
+    value = f'{value:,.2f}'
+    return value.replace(',', 'x').replace('.', ',').replace('x', '.')
```

Comment:
Found this function interesting so thought I see what it does. 

Another option is using `translate`:

```
>>> brazilian_float_trans = str.maketrans(',.', '.,')                      

>>> '{:,.2f}'.format(10000 / 3).translate(brazilian_float_trans)           
'3.333,33'

>>> '{:,.2f}'.format(10000 / 3)                                            
'3,333.33'
```

Reply:
Wow! That's way better! Refactored in 0eb34fe